### ISSUE TITLE
Quest Log: Play/Queue button is still visible for quests without voiceover

### ISSUE THREAD

**Issue by mezanddav** [mrthinger/wow-voiceover#40]
Quest link: https://www.wowhead.com/classic/quest=1048/into-the-scarlet-monastery
Quest id: 1048
Error: Sound does not exist.
Image: [https://i.kek.sh/PkYCrW3KadR.png](https://i.kek.sh/PkYCrW3KadR.png)

**Problem**: Quests without voiceover have a faulty button. The play state remains ON and it's messing up the queue.
**Solution**: To make the UI intuitive- I suggest removing the play/queue button from the Quest Log if it doesn't have a voiceover file associated. 

Details: 
WOW Classic era (Installed from CurseForge using the Curse addon manager)
Windows 10

Full list of addons:
Atlas Classic WoW
AtlasLootClassic
Auctionator
Bagnon ItemLevel
Bartender 4
Nova World Buffs
Plater nameplates
Questie
RareScanner
Routes
Spy Classic
Target Health
TomTom
WekAuras
WoW-ProGuides

**Comment by mezanddav** [comment]
If removing is not an option, here is an alternative:
Grey button out and display the error message n mouse over (Potentially better open than my original suggestion UX-wise)

**Comment by mrthinger** [comment]
we have someone working on this #36
